app/models/probate.rb

```diff
@@ -0,0 +1,5 @@
+class Probate < Base
+  attribute :kase, Boolean
```

Comment:
Could this not be name? It would match the prototype and prevent confusion sown the line
http://ministryofjustice.github.io/fr-prototypes/public/journey-7/probate.html